tr/news/_posts/2019-04-01-ruby-2-4-6-released.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: news_post
+title: "Ruby 2.4.6 Yayınlandı"
+author: "usa"
+translator: "İsmail Arılık"
+date: 2019-04-01 06:00:00 +0000
+lang: tr
+---
+
+Ruby 2.4.6 yayınlandı.
+
+Bu yayın önceki yayından sonra 20 hata düzeltmesi içerir, ve ayrıca birkaç güvenlik düzeltmesi içerir.
```

Comment:
`Bu sürüm önceki sürümden sonra 20 hata düzeltmesi içerir, ve ayrıca birkaç güvenlik düzeltmesi içerir.`

Reply:
"Yayın" diye çevirdiğim "release" kelimesi. "Version"ı "sürüm" olarak çevirmeyi tercih ettim. Ayrıca "release" kelimesinin eylem hali "yayınlamak" olarak çevriliyor bildiğim kadarıyla.

Reply:
ben 1-1 çeviri yapmak zorunda olmamalıyız diye düşünüyorum. kelime kelime çevirirsek birçok yerde tıkanırız bence. Burada bir yazılım dilinden bahsettiğimiz için ona sürüm demek daha doğru bence. basılı bir şey olsaydı bu yayın demek daha doğru olabilirdi.

Reply:
10a63e9